xml/System.Security.Claims/Claim.xml

```diff
@@ -89,9 +89,18 @@ if (null != principal)
         <Parameter Name="reader" Type="System.IO.BinaryReader" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="reader">A <see cref="T:System.IO.BinaryReader" /> pointing to a <see cref="T:System.Security.Claims.Claim" />.</param>
```

Comment:
Earlier, line 50 includes the phrase "written to HTTP the response." I think this should be "the `<xref:System.Web.HttpResponse>` object."